### PR TITLE
Add support for repmgr's `location` configuration

### PR DESCRIPTION
**Pull request by mrchrisb**
**Description of the change**

This PR adds a new environment variable, `REPMGR_NODE_LOCATION` which is used to set the `location` configuration parameter in repmgr.conf.

**Benefits**

The `location` setting is a new feature in repmgr 4 which better facilitates geographically dispersed clusters (https://repmgr.org/docs/4.2/repmgrd-network-split.html for additional details)

Setting this via an environment variable means that the user does not have to go to the additional trouble of using a fully custom configuration file just to be able to set this parameter,

**Possible drawbacks**

None identified

**Applicable issues**

None

**Additional information**

If unset, we use the string `default` as the value for the `location` parameter. This follows repmgr's documented default at https://repmgr.org/docs/4.2/repmgrd-network-split.html.

### CODE REVIEW

File: 10/debian-10/rootfs/opt/bitnami/scripts/librepmgr.sh [hunk 1]
```diff
@@ -445,6 +445,8 @@ repmgr_postgresql_configuration() {
 repmgr_generate_repmgr_config() {
     info "Preparing repmgr configuration..."
 
+    local location="${REPMGR_LOCATION:-default}"
```
Comment (miguelaeh) [inline 10/debian-10/rootfs/opt/bitnami/scripts/librepmgr.sh:448]: This should be added to the `-env.sh` file with the default value and then you can use here just `local location="${REPMGR_LOCATION}"

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for the PR!!
Please take a look at my comments

File: 10/debian-10/rootfs/opt/bitnami/scripts/librepmgr.sh (hunk 1 above)
Comment (mrchrisb) [reply 10/debian-10/rootfs/opt/bitnami/scripts/librepmgr.sh]: Yes, that is a great point! This also makes it possible to load the env var from a file for extra flexibility for users. I have added commit `bc261a2` to this effect.

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Hi @mrchrisb ,
It seems that the env var should be called `REPMGR_NODE_LOCATION` regarding the `README.md`. I already did the changes to reflect this internally but I need you to change all references from `REPMRG_LOCATION` to `REPMGR_NODE_LOCATION` in order to match what our pipeline will release

**Comment by mrchrisb** [comment]
Corrected, thanks!

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**